crates/turborepo-lib/src/turbo_json.rs

```diff
@@ -0,0 +1,18 @@
+use std::{error::Error, fs::File, io::BufReader, path::Path};
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+pub struct TurboJson {
+    pub turboVersion: String,
+}
+
+pub fn read_turbo_json<P: AsRef<Path>>(path: P) -> Result<TurboJson, Box<dyn Error>> {
+    let file = File::open(path)?;
+    let reader = BufReader::new(file);
+
+    // FIXME: serde_json doesn't support JSONC.
```

Comment:
https://github.com/serde-rs/json/blob/master/src/de.rs#L248

Anywhere you can call `parse_whitespace` in `serde_json` you could also have a comment. In a fork we could just scan to the end of and then resume eating whitespace.

Reply:
I _think_ it can be designed in a way that some automated tooling for rebasing, running a test, and then publishing it _might_ work. In other words, I _think_ the change is small enough that we can automate the fork. And then `serde_jsonc` goes up in usage to the point that eventually dtolnay reconsiders his opinion and lands the proven-to-work patch.

Reply:
I ended up doing this: https://github.com/nathanhammond/serde_jsonc/